zebra-rpc/src/methods/tests/snapshot/get_block_template_rpcs.rs

```diff
@@ -105,7 +105,8 @@ pub async fn test_responses<State, ReadState>(
         extra_coinbase_data: None,
         debug_like_zcashd: true,
         // TODO: Use default field values when optional features are enabled in tests #8183
-        ..Default::default()
```

Comment:
Is this change needed here ?

Reply:
It's just to fix `cargo clippy --no-default-features`, I think I just noticed it while I was on this branch and thought it was small and low-priority enough to include, but it doesn't have to be in this PR.